testgrid/specs/full.yaml

```diff
@@ -1371,6 +1371,28 @@
     ekco:
       version: latest
   flags: "labels=disktype=ssd,gpu=enabled"
+- name: k8s124-with-flags
+  installerSpec:
+    kubernetes:
+      version: 1.24.x
+    containerd:
+      version: latest
+    weave:
+      version: latest
+    contour:
+      version: latest
+    rook:
+      isBlockStorageEnabled: true
+      version: latest
+    registry:
+      version: latest
+    kotsadm:
+      version: latest
+    velero:
+      version: latest
+    ekco:
+      version: latest
+  flags: "labels=disktype=ssd,gpu=enabled"
```

Comment:
we could probably just update the 1.23 flags test

Reply:
fixed.